src/main/java/org/jqassistant/contrib/plugin/php/scanner/PHPFileScannerPlugin.java

```diff
@@ -5,45 +5,44 @@
  */
 package org.jqassistant.contrib.plugin.php.scanner;
 
-import java.io.BufferedReader;
-import java.io.IOException;
-import java.io.InputStreamReader;
-import java.util.ArrayList;
-import java.util.List;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
 import com.buschmais.jqassistant.core.scanner.api.Scanner;
 import com.buschmais.jqassistant.core.scanner.api.ScannerPlugin;
 import com.buschmais.jqassistant.core.scanner.api.Scope;
 import com.buschmais.jqassistant.core.store.api.Store;
 import com.buschmais.jqassistant.plugin.common.api.model.FileDescriptor;
 import com.buschmais.jqassistant.plugin.common.api.scanner.AbstractScannerPlugin;
 import com.buschmais.jqassistant.plugin.common.api.scanner.filesystem.FileResource;
-import java.io.File;
-import java.io.InputStream;
 import org.jqassistant.contrib.plugin.php.model.PHPFileDescriptor;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
 
 import static java.util.Arrays.asList;
-import org.antlr.v4.runtime.CharStreams;
-import org.antlr.v4.runtime.CommonTokenStream;
-import org.antlr.v4.runtime.Lexer;
-import org.antlr.v4.runtime.TokenStream;
-import org.antlr.v4.runtime.tree.ParseTree;
-import org.apache.commons.io.FileUtils;
-import org.jqassistant.contrib.plugin.php.PhpLexer;
-import org.jqassistant.contrib.plugin.php.PhpParser;
-import org.jqassistant.contrib.plugin.php.scanner.parser.PHPFileParser;
 
+/**
+ * PHP file scanner
+ * @author falk
+ */
 @ScannerPlugin.Requires(FileDescriptor.class)
 public class PHPFileScannerPlugin extends AbstractScannerPlugin<FileResource, PHPFileDescriptor> {
+    
     private static final Logger LOGGER = LoggerFactory.getLogger(PHPFileScannerPlugin.class);
+    
     public static final String JQASSISTANT_PLUGIN_PHP_SUFFIXES = "jqassistant.plugin.php.suffixes";
 
     private static List<String> suffixes = asList("php", "phtml");
 
-
+    /**
```

Comment:
If there is no explanation for _params_ or _return values_ you can remove the lines accordingly. 

Something like that would be useful:

`Decides if a file gets accepted by the PHP plugin. Therefore, we check if the file name ends with certain suffixes defined in <i>JQASSISTANT_PLUGIN_PHP_SUFFIXES</i>.`